pymongo/mongo_client.py

```diff
@@ -846,7 +857,11 @@ def target():
             from pymongo.encryption import _Encrypter
 
             self._encrypter = _Encrypter(self, self.__options.auto_encryption_opts)
-        self._timeout = options.timeout
+        self._timeout = self.__options.timeout
```

Comment:
I suggest moving this section out of _init_deadlockables since it only needs to happen once in `__init__`, not after a fork:
```
        if connect:
            self._get_topology()
        self._encrypter = None
        if self.__options.auto_encryption_opts:
            from pymongo.encryption import _Encrypter

            self._encrypter = _Encrypter(self, self.__options.auto_encryption_opts)
        self._timeout = options.timeout
```